app/design/frontend/UnitedArtsEducation/unitednow/Magento_Catalog/templates/product/list/items.phtml

```diff
@@ -0,0 +1,261 @@
+
+            $image = 'cart_cross_sell_products';
+            $title = __('More Choices:');
+            $items = $block->getItemCollection();
+
+            $showAddTo = true;
+            $showCart = true;
+            $templateType = \Magento\Catalog\Block\Product\ReviewRendererInterface::SHORT_VIEW;
+            $description = false;
+            $canItemsAddToCart = false;
+        }
+        break;
+
+    case 'crosssell':
+        /** @var \Magento\Catalog\Block\Product\ProductList\Crosssell $block */
+        if ($exist = count($block->getItems())) {
+            $type = 'crosssell';
+            $class = $type;
+
+            $image = 'cart_cross_sell_products';
+            $title = __('More Choices:');
+            $items = $block->getItems();
+
+            $showAddTo = true;
+            $showCart = true;
+            $templateType = \Magento\Catalog\Block\Product\ReviewRendererInterface::SHORT_VIEW;
+            $description = false;
+            $canItemsAddToCart = false;
+        }
+        break;
+
+    case 'new':
+        if ($exist = $block->getProductCollection()) {
+            $type = 'new';
+            $mode = 'grid';
+            $type = $type . ' ' . $mode;
+
+            $class = 'widget' . ' ' . $type;
+
+            $image = 'new_products_content_widget_grid';
+            $title = __('New Products');
+            $items = $exist;
+
+            $showAddTo = true;
+            $showCart = true;
+            $templateType = \Magento\Catalog\Block\Product\ReviewRendererInterface::SHORT_VIEW;
+            $description = ($mode == 'list') ? true : false;
+            $canItemsAddToCart = false;
+        }
+        break;
+
+    default:
+        $exist = null;
+}
+?>
+
+<?php if ($exist):?>
+
+<?php if ($type == 'related' || $type == 'upsell'): ?>
+<?php if ($type == 'related'): ?>
+<div class="block <?= /* @escapeNotVerified */ $class ?>" data-mage-init='{"relatedProducts":{"relatedCheckbox":".related.checkbox"}}' data-limit="<?= /* @escapeNotVerified */ $limit ?>" data-shuffle="<?= /* @escapeNotVerified */ $shuffle ?>">
+    <?php else: ?>
+    <div class="block <?= /* @escapeNotVerified */ $class ?>" data-mage-init='{"upsellProducts":{}}' data-limit="<?= /* @escapeNotVerified */ $limit ?>" data-shuffle="<?= /* @escapeNotVerified */ $shuffle ?>">
+        <?php endif; ?>
+        <?php else: ?>
+        <div class="block <?= /* @escapeNotVerified */ $class ?>">
+            <?php endif; ?>
+            <div class="block-title title">
+                <strong id="block-<?= /* @escapeNotVerified */ $class ?>-heading" role="heading" aria-level="2"><?= /* @escapeNotVerified */ $title ?></strong>
+            </div>
+            <div class="block-content content" aria-labelledby="block-<?= /* @escapeNotVerified */ $class ?>-heading">
+                <?php if ($type == 'related' && $canItemsAddToCart): ?>
+                    <div class="block-actions">
+<!--                        EDIT: Remove 'select all' option  -->
```

Comment:
This is a good start, but note that there is JS instantiating the select-all feature on line 158. You should remove this also, and you should further note that it's inside a couple of conditionals which can be simplified. With the removal of line 158, the section from 156 to 162 should be able to be logically reduced to a single conditional that simply checks `if ($type == 'upsell')` and then then outputs the code currently on line 160.
(If you have questions, let me know.)

Reply:
(As we discussed, this is actually fine as-is, since the JS is needed for other aspects of the functionality, and it's not worth pulling it forward to remove just a couple methods.)